content/vendor/guides/ci-cd-integration.md

```diff
@@ -0,0 +1,147 @@
+---
+date: "2020-02-20T00:00:00Z"
+lastmod: "2020-02-20T00:00:00Z"
+title: "CI/CD Integration"
+weight: "1007"
+---
+
+The KOTS CI/CD Integration guide is an advanced workflow that allows you to use a similar workflow to [Kots Starter GitHub Actions](https://github.com/replicatedhq/replicated-starter-kots/blob/master/.github/workflows/main.yml#L21) but for your own internal git repository and CI/CD systems like GitHub, GitLab, CircleCI, TravisCI, Jenkins, etc.
+
+It is broken into three sections:
+
+- [Environment Variables](#environment-variables)
+- [Kubernetes Objects](#kubernetes-objects)
+- [Helm Chart](#helm-chart)
+
+## Environment Variables
+
+This section will assume you already have the App slug and API token from the [Automating Your Workflow](/vendor/guides/quickstart/#automating-your-workflow) guide.
+
+- Configure the following 2 environment variables in your CI/CD configuration.
+
+- `REPLICATED_APP` with the App Slug.
+- `REPLICATED_API_TOKEN` with the API token.
+
+## Kubernetes Objects
+
+Follow the steps below if you are just working with plain-YAML kubernetes API objects.
+
+- Start from a basic Kubernetes App, which could look something like this inside the `k8s` directory.
+
+```text
+.
+└── k8s
+    ├── config-map.yaml
+    ├── deployment.yaml
+    ├── ingress.yaml
+    └── service.yaml
+```
+
+- Make a copy of [manifests](https://github.com/replicatedhq/replicated-automation/tree/master/vendor/nginx-app/manifests) directory from the KOTS starter repo and place it in your internal repo's root directory.
+
+- Replace the files in `manifests/app` with the files in `k8s`. Sub directories are supported as well, so `k8s` directory can be put anywhere inside `manifests`.
```

Comment:
I still think this isn't quite right. Do you think there's a way for me to augment my existing k8s repo without having to move my manifests around?

Reply:
Sure let me think about this. The thing here is with the Helm chart there is only 1 non-kots.io API file, the `tar.gz`. But with the normal kotsapp there are more than one non-kots app files so its a bit tricky. Lets discuss this further during the FDE standup on Wed.